generators/config/templates/default.yml

```diff
@@ -26,7 +26,10 @@
 
  # -Selenium-
  # specify one of next browsers: iexplore (ie), firefox (ff), chrome, safari
+ # NOTE: To run chrome in headless mode maximized_window option should be false,
+ # chrome version >= 59, chromedriver version >= 2.29
  selenium_browser: ff
+ chrome_headless_mode: false
```

Comment:
I do not like this option. I would introduce a new driver instead - **chrome_headless**